### DISCUSSION TITLE
cli tool for creating new icons

### DISCUSSION THREAD

**Discussion by sachinraja** [simple-icons/simple-icons#6294] [General]
Hey, just wanted to share something I made for creating new icons, [create-si](https://github.com/sachinraja/create-si). It's a cli that prompts for values and automatically handles adding the icon. It adds the `<title>` element based on the input and checks for conflicting slugs. It creates the JSON object and adds it to the data alphabetically, runs `svgo`, and runs `svglint`. I'd love any feedback on this, if you want to check it out you can run `npx create-si` in the `simple-icons` directory.

**Comment by adamrusted** [comment]
Just to add before closing, we have now implemented `get-filename` and `add-icon-data` as scripts that can be run from the project to populate the JSON. We've not automated the creation of SVGs themselves, as in order to resolve any issues brought up by maintainers - users should have at least a basic understanding of the code structure for the SVG.
